src/main/scala/com/imaginea/activegrid/core/models/AnsibleScriptEngine.scala

```diff
@@ -0,0 +1,40 @@
+package com.imaginea.activegrid.core.models
+import java.io.File
+import com.imaginea.activegrid.core.utils.{FileUtils}
+import com.typesafe.scalalogging.Logger
+import org.slf4j.LoggerFactory
+import scala.sys.process._
+import scala.util.Try
+/**
+  * Created by sivag on 23/3/17.
+  */
+class AnsibleScriptEngine(val inventory: Inventory, val workflowContext: WorkflowContext, val playBook: AnsiblePlayBook) {
+  val logger: Logger = Logger(LoggerFactory.getLogger("AnsibleScriptEngine"))
+  /**
+    * Creates process to run playbook and starts the process.
+    * Writes output to a logfile.
+    * @return true if process started successfully.
+    */
+  def executeScript(): Try[Boolean] = {
```

Comment:
Method will be called from  '**def executePlayBook() : Try[Boolean]**' method of another object.